src/entry_points/eest_utils.py

```diff
@@ -0,0 +1,87 @@
+"""
+CLI interface for common EEST shortcuts and utilities.
+"""
+
+import os
+import shutil
+import subprocess
+
+import click
+
+from logger import setup_logger
+
+logger = setup_logger(__name__)
+
+
+@click.group()
+def eest():
+    """
+    EEST commands for common repo utilities.
+    """
+    pass
+
+
+@eest.command()
+def init():
+    """
+    Initializes and installs all required packages for the repository:
+    ```
+    python3 -m venv ./venv/
+    source ./venv/bin/activate
+    pip install -e '.[docs,lint,test]'
+    ```
+    """
+    logger.info("Creating a virtual environment: `python3 -m venv ./venv/`")
+    subprocess.run(["python3", "-m", "venv", "./venv/"], check=True)
```

Comment:
```suggestion
    subprocess.run([sys.executable, "-m", "venv", os.path.join(".", "venv")], check=True)
```
Use `sys.executable` to reuse the python binary that the user used to run the script.

Reply:
[758b154](https://github.com/ethereum/execution-spec-tests/pull/461/commits/758b1540c673ed918ded01dec5d4364382ab3e12)